### PR TITLE
Unify page titles

### PR DESCRIPTION
**Pull request by hikes101**
I added some titles and adjusted some content. We can discuss about the final page layout later. Especially the blog page needs to be redone, but that will be a separate pr.

Let me know if you have any requests regarding changes.

### CODE REVIEW

**Comment by hikes101** [comment]
[Unified titles](https://trello.com/c/Ksw2u71I/111-unified-titles)

File: src/views/projects.hbs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{{> main/title title="Projekte & mehr"}}
+{{> main/title title="Aktuelle Projekte & mehr"}}
```
Comment (pheeria) [inline src/views/projects.hbs]: Let's just make only _Projekte_.

File: src/views/partials/main/title.hbs [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <div class="container mb-2">
   <div class="row">
     <div class="col-12">
-      <p class="h1 font-italic my-4">{{title}}</p>
+      <p class="h1 my-4">{{title}}</p>
       <p class="lead">{{subtitle}}</p>
       <hr>
```
Comment (jarjan) [inline src/views/partials/main/title.hbs]: Maybe remove this `<hr>` and add `border-bottom`?
https://getbootstrap.com/docs/4.3/utilities/borders/#additive

Comment (hikes101) [reply src/views/partials/main/title.hbs]: Fixed it

**Pull request merged**